### PR TITLE
Update event source

### PR DESCRIPTION
**Pull request by mischah**
2019 Update Regarding #409

### CODE REVIEW

File: posts/Eventsource.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
+tags: polyfill
 kind: api
-polyfillurls:[EventSource.js](https://github.com/remy/polyfills/blob/master/EventSource.js), [jQuery.eventsource](https://github.com/rwldrn/jquery.eventsource)
+polyfillurls:[EventSource polyfill](https://github.com/Yaffle/EventSource)
 
-[Server-Sent Events](http://html5doctor.com/server-sent-events/) are real-time events emitted by the server and received by the browser. EventSource.js provides a base fallback and is not dependent on jQuery.
+Server-sent events are real-time events emitted by the server and received by the browser.
+The main different to WebSockets is that they work over regular HTTP and can therefore be replicated entirely using JavaScript. 
+
+Event Source isn’t available in any version of Internet Explorer and still under [consideration for Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/serversenteventseventsource/?q=EventSource).
```
Comment (coliff) [inline posts/Eventsource.md]: Should Event Source be EventSource here?

File: posts/Eventsource.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
 status: use
-tags: fallback
+tags: polyfill
 kind: api
-polyfillurls:[EventSource.js](https://github.com/remy/polyfills/blob/master/EventSource.js), [jQuery.eventsource](https://github.com/rwldrn/jquery.eventsource)
+polyfillurls:[EventSource polyfill](https://github.com/Yaffle/EventSource)
 
-[Server-Sent Events](http://html5doctor.com/server-sent-events/) are real-time events emitted by the server and received by the browser. EventSource.js provides a base fallback and is not dependent on jQuery.
+Server-sent events are real-time events emitted by the server and received by the browser.
+The main different to WebSockets is that they work over regular HTTP and can therefore be replicated entirely using JavaScript.
```
Comment (coliff) [inline posts/Eventsource.md]: I think this should be;
> The main difference to WebSockets...

File: posts/Eventsource.md (hunk 1 above)
Comment (mischah) [reply posts/Eventsource.md]: Your right.

File: posts/Eventsource.md (hunk 2 above)
Comment (mischah) [reply posts/Eventsource.md]: Jup.

**Comment by mischah** [comment]
Thanks for catching those. Updated and did a force push.

**Pull request merged**